### ISSUE TITLE
Accessibility: Keyboard Shortcuts Lacking

### ISSUE THREAD

**Issue by GfEW** [orgzly-revived/orgzly-android-revived#345]
- [x] I have searched for existing issues that may be the same as or related to mine.

Please complete the following information:

    Device: diverse
    Android Version: 9 to 11
    Orgzly Revived Version: 1.8.28
    Sync repository type: any

### The Problem

Although android is commonly considered touch centric, there are various reasons people use external keyboards with their android devices, be it occasionally or even primarily.

As of 1.8.28, Orgzly Revived fails to provide quick, reliable keyboard access (as opposed to tabbing around) even to the most basic, frequently needed actions. 

### To Reproduce

**To give an arbitrary example (out of many):** 
1. Connect an external keyboard to your android device, and open Orgzly Revived there.
2. Enter any note view by keyboard navigation.
3. Try to start editing the note body, by keyboard.
4. Try to save changes and leave edit mode again, by keyboard.

### What does happen?

You likely don't even get to 4. because Orgzly won't let you enter edit mode by keyboard in 3.

### What should happen?

You should be easily able to edit the note, save your changes and close it again, by keyboard.

**So, below is a Starter 
with some ideas of sensible shortcuts:**

### In any lists of items (notebooks, notes, search results, ...):
* move the item selector one by one: `Up`/`Down` _(this works already)_
* move the item selector more quickly: `Home`/`PgUp`/`PgDown`/`End`
* select item(s): `Shift+Up`/`Shift+Down`, or select all: `Ctrl+A`
  * cut/copy/paste: `Ctrl+X`/`Ctrl+C`/`Ctrl+V` _(if applicable)_
  * delete: `Del` _(with confirmation)_
* open and focus the search widget: `Ctrl+F` _(keeping the search term in place, if any)_
* open and focus the navigation drawer: `Ctrl+O` _(closing it by `Back` (i. e. `Meta-Backspace`) works already)
* sync: `Ctrl+S` (or `Ctrl+R`? - `F5` is also common but not easily availably on all keyboards)

#### In the list of notebooks:
* create new notebook: `Ctrl+Shift+N` or `Ctrl+N`

#### In a list of notes, i. e. a specific notebook:
* create new note: `Ctrl+N` or `+`
* with selected note(s)
  * assign a state: `i`
  * set priority: `p`
  * schedule: `s`
  * move selected note(s): `Alt+Up`/`Alt+Down`/`Alt+Left`/`Alt+Right`
* fold/unfold note: `Left`/`Right`
* open and focus search widget: `Ctrl+F`
* open the navigation drawer: `Ctrl+O`
 
#### In a specific note, view mode:
* edit selected field: `Enter` _(as if the field was tapped)_

#### In a specific note, edit mode:
* with selected text, style it bold/code/italic/underlined: `Ctrl+B`/`Ctrl+E`/`Ctrl+I`/`Ctrl+U`
* confirm changes, leave note: `Ctrl+Enter` _(as if the edit confirmation checkmark was tapped)_

These suggestions are obviously open to discussion and don't try to emulate emacs defaults, as we`re in CUA world anyway.

**Comment by GfEW** [comment]
Here's a starter with ideas of sensible shortcuts for currently inaccessible actions.
### 1. In any lists of items 
#### (i. e. notebooks, notes, search results, ...):
* move through long lists more quickly than single-step Up/Down: `Home`/`PgUp`/`PgDown`/`End`
* select/deselect item(s): `Shift+Up`/`Shift+Down`, or select all: `Ctrl+A`
* with selection
  * cut/copy/paste: `Ctrl+X`/`Ctrl+C`/`Ctrl+V` _(if applicable)_
  * delete: `Del` _(with confirmation)_
* open and focus the search widget: `Ctrl+F` _(keeping the search term in place, if any)_
* open and focus the navigation drawer: `Ctrl+O` _(closing it by `Back` (i. e. `Meta-Backspace`) works already)
* sync: `Ctrl+S` (or `Ctrl+R`? - `F5` is also common but not easily availably on all keyboards)

### 2. In specific lists
#### 2.1. In the list of notebooks:
* create new notebook: `Ctrl+Shift+N` or `Ctrl+N`

#### 2.2. In a list of notes, i. e. a specific notebook:
* create new note: `Ctrl+N` or `+`
* with selected note(s)
  * assign a state: `i`
  * set priority: `p`
  * schedule: `s`
  * move selected note(s): `Alt+Up`/`Alt+Down`/`Alt+Left`/`Alt+Right`
* fold/unfold note: `Left`/`Right` _(together with `Up`/`Down`, this allows for very quick navigation across notes, akin to "elinks mode" e. g. in mc)_
* open and focus search widget: `Ctrl+F`
* open the navigation drawer: `Ctrl+O`
 
#### 2.3. In a specific note, view mode:
* edit selected field: `Enter` _(as if the field was tapped)_

#### 2.4. In a specific note, edit mode:
* with selected text
  * style it bold/code/italic/underlined: `Ctrl+B`/`Ctrl+E`/`Ctrl+I`/`Ctrl+U`
* confirm changes, leave note: `Ctrl+Enter` _(as if the edit confirmation checkmark was tapped)_

These suggestions are obviously open to discussion and don't try to emulate emacs defaults, as we`re in CUA world anyway.
